recipes/python/backup-restore/workload_oracle.py

```diff
@@ -0,0 +1,93 @@
+"""
+The library contain functions related to NetBackup Oracle workload
+"""
+
+## The script can be run with Python 3.6 or higher version.
+
+import common
+import json
+import os
+
+headers = {"Content-Type" : "application/vnd.netbackup+json;version=4.0"}
+
+# Get oracle_asset info
+def get_oracle_asset_info(baseurl, token, asset_type, display_name):
+    """ This function return the asset info """
+    print(f"Get client asset info for Oracle type :[asset_type]")
+    headers.update({'Authorization': token})
+    if (asset_type == "DATABASE"):
+        url = f"{baseurl}asset-service/workloads/oracle/assets?"\
+            f"filter=assetType eq '{asset_type}' and databaseUniqueName eq '{display_name}'"
+    else:
+        url = f"{baseurl}asset-service/workloads/oracle/assets?"\
+            f"filter=assetType eq '{asset_type}' and displayName eq '{display_name}'"
+    status_code, response_text = common.rest_request('GET', url, headers)
+    common.validate_response(status_code, 200, response_text)
+
+    asset_id = response_text['data'][0]['id']
+
+    print(f"Client asset Id:[{asset_id}]")
```

Comment:
should we make it guaranteed that this print method runs only if the asset_id is populated after getting a response? So that we can avoid asset_id being null?

Reply:
I am just curious why we are not using async functions here. Since we have HTTP request or RESTAPI calls, each execution may not happen synchronously.